index.rst

```diff
@@ -48,12 +48,336 @@
+   +---------------------------------------------------+-------------+-------------------------------------------------------------------------------+
+   | [<instrument>/]runs/<target>/<release>/<ticket>/* | unspecified | Private intermediates of processing data <target> with <release> on <ticket>. |
+   +---------------------------------------------------+-------------+-------------------------------------------------------------------------------+
+   | refcats                                           | CHAINED     | All reference catalogs (distinguished by dataset type).                       |
+   +---------------------------------------------------+-------------+-------------------------------------------------------------------------------+
+   | refcats/<ticket>                                  | RUN         | One reference catalog, ingested and sharded on <ticket>.                      |
+   +---------------------------------------------------+-------------+-------------------------------------------------------------------------------+
+   | skymaps                                           | RUN         | All skymap definition datasets (distinguished by data ID).                    |
+   +---------------------------------------------------+-------------+-------------------------------------------------------------------------------+
+   | u/<user>/*                                        | unspecified | Experimental/development processing by <user>.                                |
+   +---------------------------------------------------+-------------+-------------------------------------------------------------------------------+
+
+.. _daf_butler documentation: https://pipelines.lsst.io/v/weekly/modules/lsst.daf.butler/organizing.html
+
+.. _collections-per-instrument:
+
+Per-instrument collections
+--------------------------
+
+Raw and calibration data associated with a particular instrument is organized into collections that start with the "short" instrument name, e.g. "HSC" or "LSSTCam-imSim", followed by a slash.
+The naming conventions for these collections are codified by the `lsst.obs.base.Instrument`_ class's ``make*Name`` methods.
+The highest-level collections are always defined as ``CHAINED`` collection "pointers" to other versioned collections.
+
+In the case of raw data, we propose three levels of collections:
+
+ - ``<instrument>/raw/all``: the ``RUN`` collection into which all raws for that instrument are originally ingest.
+ - ``<instrument>/raw/good/<ticket>``: a ``TAGGED`` collection containing a curated subset of all raws that do not contain problems (e.g. tracking issues, airplanes, etc.), named according to the ticket (e.g. ``DM-98765``) on which the curation work was done.
+ - ``<instrument>/raw/good``: a ``CHAINED`` collection that points to the current-best ``*/good/<ticket>`` collection for this instrument.
+
```

Comment:
Apologies if I've missed it, but perhaps a reminder somewhere that rerunning with input collections with "good" in the name may not reproduce same results (as datasets could be added or removed from those collections).  Not specifically worried about the immediate users as much as looking forward to users new to LSST.